.chloggen/fix_azure-monitor-receiver-time-grain.yaml

```
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: azuremonitorreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Azure Monitor receiver doesn't honor time grain"
```

```suggestion
note: "Fix bug where the time grain wasn't honored"
```
Nit: Just want to make it clear this bug is fixed now.